core/dbt/contracts/graph/manifest.py

```diff
@@ -567,7 +567,6 @@ class Manifest(MacroMethods, DataClassMessagePackMixin, dbtClassMixin):
     # Moved from the ParseResult object
     source_patches: MutableMapping[SourceKey, SourcePatch] = field(default_factory=dict)
     # following is from ParseResult
-    _disabled: MutableMapping[str, List[CompileResultNode]] = field(default_factory=dict)
```

Comment:
@jtcohen6  Disabled does show up in manifest.json, so this would be a manifest schema change. Do you agree with what I propose here? If we're going to do it, for 1.0 is the right time.

Reply:
Your proposal makes sense to me! We should feel well within our rights to make these kind of schema changes ahead of 1.0